server/main.go

```diff
@@ -40,7 +40,7 @@ func init() {
 	db.InitMysqlDB()
 	db.InitRedisDB()
 	client.InitMinio()
-	client.InitMail()
+	// client.InitMail()
 	hooks.InitHooks()
```

Comment:
if you don't need it just delete it

Reply:
It based on SMTP service with third-party, I annotate it to skip mail check in  openkf server start stage and easy to develop, we will enable it in release version.